src/libs/SidebarUtils.ts

```diff
@@ -278,14 +277,15 @@ function getOptionData(
     result.isThread = ReportUtils.isChatThread(report);
     result.isChatRoom = ReportUtils.isChatRoom(report);
     result.isTaskReport = ReportUtils.isTaskReport(report);
-    result.parentReportAction = parentReportAction;
+    result.parentReportAction = parentReportAction ?? undefined;
```

Comment:
Let's better update `OptionData` [type](https://github.com/Expensify/App/blob/main/src/libs/ReportUtils.ts#L338) to have `parentReportAction?: OnyxEntry<ReportAction>;`
```suggestion
    result.parentReportAction = parentReportAction;
```